source/accounts/index.rst

```diff
@@ -324,31 +324,137 @@ RSA software tokens provide two factor authentication (2FA) for NOAA RDHPCS syst
      If you don’t have a smartphone, you can request an RSA hardware token. The activation process is found here: `New User Activation <https://rdhpcs-common-docs.rdhpcs.noaa.gov/wiki/index.php/New_User_Activation#RSA_Token_Activation>`_.  RSA software tokens are preferred.
 
 
-CAC Access
-----------
+<<<<<<< Updated upstream
+Accessing the RDHPCS Systems:  CAC Access
+*****************************************
+=======
+Accessing the RDHPCS Systems
+============================
+>>>>>>> Stashed changes
 
 The Common Access Card (CAC), is the preferred means of access to RDHPCS resources for both Web and SSH access. To obtain a CAC, work with your local admin services team as they need to start the application process.  Some labs can issue CACs on-site, otherwise you will have to visit a RAPIDS site. The site locator website is `ID Card Office Online <https://idco.dmdc.osd.mil/idco/locator>`_.  SSH logins with a CAC require additional software.
 
-
-Accessing the RDHPCS Systems
-----------------------------
-
 .. NOTE::
 
    To access a system, you must be on a project assigned to that system.
```

Comment:
"Visit the RDHPCS Account Information Management System (AIM) at https://aim.rdhpcs.noaa.gov to view your HPCS account information and request access as appropriate."